README.md

```diff
@@ -94,15 +91,27 @@ core.write_8(0x2000_0000, &buff)?;
 
 ### I need help!
 
-Don't hesitate to [file an issue](https://github.com/probe-rs/probe-rs/issues/new), ask questions on [matrix](https://matrix.to/#/!vhKMWjizPZBgKeknOo:matrix.org?via=matrix.org&via=spodeli.org), or contact [@Yatekii](https://github.com/Yatekii) by e-mail.
+Don't hesitate to [file an issue](https://github.com/probe-rs/probe-rs/issues/new), ask questions on [Matrix](https://matrix.to/#/#probe-rs:matrix.org), or contact [@Yatekii](https://github.com/Yatekii) via e-mail.
 
 ### How can I help?
 
 Please have a look at the issues or open one if you feel that something is needed.
 
-Any contibutions are very welcome!
+Any code contibutions are very welcome!
```

Comment:
I think other contributions (e.g. documentation or testing) are welcome as well, so I would not mention code here.
```suggestion
Any contributions are very welcome!
```